pulp_ansible/tests/functional/utils.py

```diff
@@ -33,8 +33,8 @@
     ApiClient as AnsibleApiClient,
     ContentCollectionVersionsApi,
     DistributionsAnsibleApi,
-    PulpAnsibleGalaxyApiCollectionsApi,
-    PulpAnsibleGalaxyApiV3VersionsApi,
+    PulpAnsibleApiV3CollectionsApi,
```

Comment:
I keep trying to come up with a way to make this name shorter and simpler. I guess I don't really care what the names are as long as they are clear. @fao89 any thoughts on this?

Reply:
you can pick the name you want by declaring `pulp_tag_name`